aptos-move/move-examples/hello_blockchain/Move.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "Examples"
+version = "0.0.0"
+
+[addresses]
+HelloBlockchain = "66301a58029b93bb46e93d344a79d5673a62c81f94a829b93d0e548a19269fdf"
```

Comment:
can you set this to "_" like you did for message board? and maybe update the instructions so folks can compile this more easily

Reply:
modified